src/bundler.ts

```diff
@@ -42,7 +42,7 @@ export async function extendBundler({ options: nuxtOptions }: I18nNuxtContext, n
       compositionOnly: nuxtOptions.bundle.compositionOnly,
       onlyLocales: nuxtOptions.bundle.onlyLocales,
       dropMessageCompiler: nuxtOptions.bundle.dropMessageCompiler,
-      optimizeTranslationDirective: true,
+      optimizeTranslationDirective: false,
```

Comment:
shouldn't we add a new module option (disabled by default)? Maybe we should add this in a new PR...

Reply:
Oops I didn't mean to push that, that's a remnant of testing other things. 

Ideally we can fix the issue that's forcing projects to have this disabled, if I recall correctly we need this enabled for `v-t` to work in SSR (https://github.com/nuxt-modules/i18n/pull/3014), so disabling it is not desirable.